tests/HUBCollectionViewLayoutTests.m

```diff
@@ -339,17 +339,17 @@ - (void)testProposedContentOffsetAfterRecomputing
     NSRange const addedRange = NSMakeRange(0, 2);
     NSIndexSet * const addedIndices = [NSIndexSet indexSetWithIndexesInRange:addedRange];
 
-    [removedIndices enumerateIndexesUsingBlock:^(NSUInteger idx, BOOL *stop) {
-        [self removeBodyComponentAtIndex:idx];
-        if (idx <= currentIndex) {
-            deletionHeight += self.fullWidthComponent.preferredViewSize.height;
+    [addedIndices enumerateIndexesUsingBlock:^(NSUInteger idx, BOOL *stop) {
+        [self addBodyComponentWithIdentifier:self.fullWidthComponentIdentifier preferredIndex:idx];
+        if (idx < currentIndex) {
+            insertionHeight += self.fullWidthComponent.preferredViewSize.height;
         }
     }];
 
-    [addedIndices enumerateIndexesUsingBlock:^(NSUInteger idx, BOOL *stop) {
-        [self addBodyComponentWithIdentifier:self.fullWidthComponentIdentifier preferredIndex:idx];
+    [removedIndices enumerateIndexesUsingBlock:^(NSUInteger idx, BOOL *stop) {
```

Comment:
Could you just give some quick context as to why these tests had to be changed? 😄